CITATIONS.md

```diff
@@ -18,6 +18,14 @@
 
   > Ewels P, Magnusson M, Lundin S, Käller M. MultiQC: summarize analysis results for multiple tools and samples in a single report. Bioinformatics. 2016 Oct 1;32(19):3047-8. doi: 10.1093/bioinformatics/btw354. Epub 2016 Jun 16. PubMed PMID: 27312411; PubMed Central PMCID: PMC5039924.
 
+- [Simpleaf](https://doi.org/10.1093/bioinformatics/btad614)
+
+  > He, D., Patro, R. simpleaf: a simple, flexible, and scalable framework for single-cell data processing using alevin-fry, Bioinformatics 39, 10 (2023).
+
+* [Alevin-fry](https://doi.org/10.1038/s41592-022-01408-3)
+
+  > He, D., Zakeri, M., Sarkar, H. et al. Alevin-fry unlocks rapid, accurate and memory-frugal quantification of single-cell RNA-seq data. Nat Methods 19, 316–322 (2022).
+
 * [Alevin](https://doi.org/10.1186/s13059-019-1670-y)
```

Comment:
Is alevin still relevant here? Is `alevin-fry` totally different or it uses alevin?

Reply:
perfect